src/store/getters/state-getters.ts

```diff
@@ -104,7 +104,7 @@ export const getNameRequestNumber = (state: any): string => {
  * Returns the approved name of a name request.
  */
 export const getApprovedName = (state: any): string => {
-  return state.stateModel.nameRequest.details.approvedName
+  return state.stateModel.nameRequest.legalName
```

Comment:
I think, at this point, the NR is no longer useful. You should be able to get the Legal Name from the business data instead.
(I'm unsure what the code will look like if we associated a new NR!)

Reply:
Ahh right, thanks. Will look at this!